### ISSUE TITLE
Why AuthorizationResult.cs has an internal constructor?

### ISSUE THREAD

**Issue by evaldas-raisutis** [nozzlegear/ShopifySharp#564]
This makes mocking unnecessarily difficult. Is it a problem for it to have a public constructor? Or a an constructor which takes parameters?

**Comment by nozzlegear** [comment]
Good catch! I'm not sure why it's an internal constructor, I think we can easily make it public or at least protected without breaking anything.

--
Josh Harms
